test/integration/short/timeout-tests.js

```diff
@@ -7,6 +7,7 @@ var Connection = require('../../../lib/connection');
 var utils = require('../../../lib/utils');
 var types = require('../../../lib/types');
 var errors = require('../../../lib/errors');
+var ExecutionProfile = require('../../../lib/execution-profile').ExecutionProfile;
 
 describe('client read timeouts', function () {
   this.timeout(120000);
```

Comment:
It looks like there are more test regressions that are unrelated to this - digging into it.